attributes/network/3_kvstore.adoc

```diff
@@ -8,35 +8,40 @@ It exposes the following endpoints:
 
 - <<_info, `kvstore.info`>>.
     Returns information about this kvstore.
+- <<_get, `kvstore.get`>>.
+    Returns the value stored at a specific key.
 - <<_put, `kvstore.put`>>.
-    Returns the balance of the account specified (or the sender).
-- <<_get, `kvstore.get`>>. Creates new tokens and add them to an account.
+    Sets the value at the specific key.
+- <<_delete, `kvstore.delete`>>.
+    Delete the value stored at a specific key.
 
 == Errors
-The following error codes are reserved for ledger servers:
+The following error codes are reserved for kvstore servers:
 
-- 20001, `u(symbol)`.
-    A symbol name is unknown.
-- 20002, `unauthorized`.
-    A user is not authorized to make this transaction.
-- 20003, `insufficient_funds`.
-    An account does not have enough funds for a transaction.
+- 30001, `permission_denied`.
+    A user do not have the authorization to modify this key.
+- 30002, `invalid_initial_state(expected, actual)`.
+    Invalid initial state hash.
 
 == Keys
 Keys are any valid byte string, including NULL characters.
-Keys that don't exist should return
+Keys that don't exist should return an empty value.
```

Comment:
Should probably error out instead. What's your rationale for returning empty instead of `ManyError::NotFound`?

Reply:
I will change it to an error.